core/src/components/segment/test/basic/index.html

```diff
@@ -229,6 +232,14 @@
           }
         }
 
+        function toggleSwipeGesture() {
+          const ionSegmentElement = document.querySelectorAll('ion-segment');
+          for (var i = 0; i < ionSegmentElement.length; i++) {
+            const swipeGesture = ionSegmentElement[i].swipeGesture === false ? false : true;
+            ionSegmentElement[i].swipeGesture = swipeGesture;
```

Comment:
```suggestion
            ionSegmentElement[i].swipeGesture = !ionSegmentElement[i].swipeGesture;
```

I noticed this wasn't toggling properly, so we can just do this instead.